pkg/harvester/models/harvester/node.js

```diff
@@ -82,6 +83,24 @@ export default class HciNode extends HarvesterResource {
       total:   1
     };
 
+    const promptRemove = {
+      action:     'promptRemove',
+      altAction:  'remove',
+      label:      this.t('action.remove'),
+      icon:       'icon icon-trash',
+      bulkable:   true,
+      enabled:    this.canDelete,
+      bulkAction: 'promptRemove',
+      weight:     -10,
+    };
+
+    const out = super._availableActions;
+    const remove = out.findIndex(a => a.action === 'promptRemove');
+
+    if (remove > -1) {
+      out.splice(remove, 1);
+    }
+
```

Comment:
I checked the resource-class.js, I found there is a same object.

```js
{
  action:     'promptRemove',
  altAction:  'remove',
  label:      this.t('action.remove'),
  icon:       'icon icon-trash',
  bulkable:   true,
  enabled:    this.canDelete,
  bulkAction: 'promptRemove',
  weight:     -10, // Delete always goes last
}
```

Just make sure, the reason you need to remove with `splice`, is it because the `this` scope ?

Reply:
Thanks @Yu-Jack , you are right, this can be removed.